src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/asserter/DomainEventAsserter.kt

```diff
@@ -25,6 +25,18 @@ class DomainEventAsserter(
     ).hasSize(expectedCount)
   }
 
+  fun assertHasZeroDomainEventsStoredOfEventType(applicationId: UUID, eventType: DomainEventType) =
+    assertDomainEventStoreCount(applicationId, eventType, expectedCount = 0)
```

Comment:
i can see why you're not reusing the existing assertDomainEventsOfTypeStored function because that's deliberately filtering out events with a space booking id. I'm going to create a ticket to remove that - i suspect it was a 'quick fix' to stop a bunch of tests failing but the behaviour is quite opaque

Reply:
i've created APS-1517